raiden-dapp/src/components/GeneralMenu.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="general-screen-menu">
+    <v-row class="general-screen-menu__account-details no-gutter" no-gutters>
+      <v-col cols="12">
+        <div class="general-screen-menu__account-details__title">
+          {{ $t('general-menu.account-details') }}
+        </div>
+      </v-col>
+      <v-col cols="2">
+        <span class="general-screen-menu__account-details__address">
+          {{ $t('general-menu.address') }}
+        </span>
+      </v-col>
+      <v-col cols="10">
+        <span class="general-screen-menu__account-details__address">
+          <address-display :address="defaultAccount" :full-address="true" />
```

Comment:
```suggestion
          <address-display :address="defaultAccount" full-address />
```

Reply:
I can't get this change to work?

Reply:
@taleldayekh check the address display component if the full-address property `@Prop(...)` doesn't have `type: Boolean` you should add it

Reply:
Thank you @kelsos that solved it.